arches/app/models/resource.py

```diff
@@ -228,7 +226,7 @@ def bulk_save(resources, transaction_id=None):
         se.bulk_index(documents)
         se.bulk_index(term_list)
 
-    def index(self):
+    def index(self, context=None):
```

Comment:
It would be nice to add a docstring to methods that now take in a context parameter describing what can be passed in and maybe what it's used for.
EG:
```
Keyword Arguments:
context -- defaults to False, set to True to return the raw object saved in the db 
```

Reply:
I've added some docstrings for `context`